docs/platforms/flutter/features/index.mdx

```diff
@@ -45,3 +45,14 @@ Sentry supports Flutter Web as well, with the following limitations:
 - Offline caching isn't supported for exceptions.
 - [Release Health](/product/releases/health/) isn't supported.
 - Device context data isn't as rich on the web as it is on mobile.
+
+**Desktop Limitations:**
+
+Sentry supports Flutter on Linux and Windows as well, with the following limitations:
+
+- No native crash support on Windows
+- No support for [Release Health](/product/releases/health/)
+- Device context data is not as rich as on Android, iOS & macOS
```

Comment:
```suggestion
- Native crashes are not supported on Windows.
- [Release Health](/product/releases/health/) isn't supported.
- Device context data isn't as rich on the web as it is on mobile.
```

Reply:
I think `on the web` should be `on Linux and Windows`